adapters/flipp/flipp.go

```diff
@@ -59,12 +63,12 @@ func (a *adapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.E
 		adapterRequests = append(adapterRequests, adapterReq)
 	}
 	if len(adapterRequests) == 0 {
-		return nil, append(errors, fmt.Errorf("adapterRequest is empty"))
```

Comment:
There's no need for `fmt.Errorf` with a fixed string. Hoisted the error to a top level variable to be more sensitive to allocations.